ggml-opencl.cpp

```diff
@@ -862,42 +985,46 @@ static void ggml_cl_mul_mat_q_f32(const ggml_tensor * src0, const ggml_tensor *
 
     for (int64_t i03 = 0; i03 < ne03; i03++) {
         for (int64_t i02 = 0; i02 < ne02; i02++) {
-            cl_event ev_sgemm;
+            size_t ev_idx = 0;
+            std::vector<cl_event> events;
```

Comment:
suggest move this out of loop and give a reasonable initial size. You can use clear in the inner-loop